.github/dependabot.yml

```
@@ -49,4 +59,6 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+    time: "01:00"
```

Maybe have the GHA runs later?
```suggestion
    time: "02:00"
```

---

sure